triggermesh/knative-local-registry
# Use a service instead of localhost + aliases

Remaining work:
 * Test in GKE, with and without Alias IPs
 * Automation (see ./knative-registry-operator/README.md) will be a separate PR
 * Remove localhost proxy
   - Could be more self-contained that the service-based only if sidecars were supported with init containers
 * Remove docs on the old alias
